gc/base/ParallelDispatcher.cpp

```diff
@@ -757,4 +754,61 @@ MM_ParallelDispatcher::expandThreadPool(MM_EnvironmentBase *env)
 
 	return result;
 }
+
+bool
+MM_ParallelDispatcher::reinitialize(MM_EnvironmentBase *env, uintptr_t newPoolSize)
+{
+	 if (!_extensions->isStandardGC()) {
+		  /* Dispatcher reinitialization is only supported for standard collectors. */
+		 Assert_MM_true(newPoolSize <= _poolMaxCapacity);
+	 } else if (newPoolSize > _poolMaxCapacity) {
+		/* Re-size/allocate the dispatcher tables: _threadTable, _statusTable & _taskTable. */
+		 omrthread_t *threadTableTemp;
+		 uintptr_t *statusTableTemp;
+		 MM_Task **taskTableTemp;
```

Comment:
to comply with coding standards, into these with null, or move the declaration later at the time they are first initialized/used